Makefile

```diff
@@ -1,3 +1,6 @@
+
```

Comment:
@kushthedude the workflow for each adapter is consistent. Is the Makefile for each adapter consistent as well?

Reply:
Workflow for each adapter is not consistent, only build and release flows are consistent for each adapter.